docs/core/config/mailbox/mail_location.md

```diff
@@ -297,6 +297,7 @@ export USERDB_KEYS="$USERDB_KEYS mail"
 
 exec "$@"
 ```
+:::
```

Comment:
Not sure what these `:::` additions are where there is no corresponding `::: code-group`